### ISSUE TITLE
No audio playback

### ISSUE THREAD

**Issue by davidgbarron** [microdee/Uranium#9]
Hey guys, I'm aware the USoundWave hasn't been implemented yet. But is there something we can do about the no audio playback atm? for example: a youtube video. Everything works fine except the audio - I do notice a slight blip of audio for a split second before it dies again upon loading a video. I've tried debugging it and when I pause the vs debugger for around 10 seconds and subsequently resume - I can hear the Audio from the video playback with no problems, It's hit or miss though. I've been looking into the AudioHandler and possible error messages.  But I was hoping to see if any of you might have some idea about what might be causing this. I have ideas, but I want to see if I could get some direction that would narrow my list of ideas down. Thanks in Advance.

**Comment by microdee** [comment]
Ah man this plugin haunts my backlog for so long. I had some plans for it but it fell completely off the priority shelf. I have to disappoint you there but I don't have the capacity right now to help you out there with this :( Sorry about the inconveniences

**Comment by davidgbarron** [comment]
Hey there, no worries. I appreciate the reply. I did a bit of research into the CEF classes myself and I figured out a solution. I got sound working and everything is flawless now.  It came down to the CefAudioHandler's Interface missing a function. As soon as I implemented this into the code It worked, and in hindsight it makes total sense why it wasn't working with-out it. I can go into detail if you have plans to ever continue work on this plugin. I've also created a solution for a separate issue in the form of an editor module.  I've mentioned it in a different issue (#7, I believe), It allows the user to shut down the CEFSubprocess from an Editor button. This way, devs can shut down the Process and subsequently package their projects with-out errors. It beats using the command line every-time they'd want to package. I can also provide that if you're interested. All that being said, I do thank you for the work you put into this plugin. As it's something I needed for an uber important project at my company. Also, Kudos on the code organization, made it simple to navigate and read. I've followed suit in my implementation of the "Editor Module" I created for the previous solution - in case it were ever considered for implementation in an update of Uranium. Thanks again.

David.
